operationcode_bot.rb

```diff
@@ -66,6 +66,16 @@ def team_join(data, token: nil)
   empty_response
 end
 
+def member_joined_channel(data, token: nil)
+  logger.info "Member joined channel: #{data}"
+
+  if data['event']['channel'] == Event::MENTORS_INTERNAL_CHANNEL
```

Comment:
I'm not clear on how we want the behavior of this `member_joined_channel` flow to be.  For example, if this conditional is **not** met, do we still want to log this `logger.info "Member joined channel: #{data}"`, and return `empty_response`?

Or do we want to do one/neither of those things, if the conditional is met?  Also, do we want to log a different message, and return some else, if the conditional is not met?

I'm not clear on the business logic behind this, to be able to provide a recommendation.  Just food for thought.